### ISSUE TITLE
Spec has zombie-life on /TR

### ISSUE THREAD

**Issue by gkellogg** [speced/respec#4251]
## Description of problem

**URL to affected spec or repo:**

https://github.com/w3c/json-ld-bp

**What happened (e.g., it crashed)?:**

Spec shows previousVersion on /TR when never published.

**Expected behavior (e.g., it shouldn't crash):**

It shouldn't have have a previousVersion, and shouldn't show on /TR at all.

**Optional, steps to reproduce:**

I've created a PR to change status to "unofficial": https://github.com/w3c/json-ld-bp/pull/44.
You can see through the preview that This version is on /TR, as is the Latest published version.
Interestingly, I've even comment out the `shortName` from respecConfig, it still wants to publish in the same place.

In fact, this was previously marked as a "NOTE", but the JSON-LD WG never actually got around to agreeing to publish it.

The ReSpec Errors say "Ask a W3C Team Member for a W3C URL where the report can be published and change [latestVersion](https://respec.org/docs/#latestVersion) to something else", but it has no `latestVersion`, and @iherman doesn't understand this either.

      var respecConfig = {
        localBiblio: jsonld.localBiblio,
        specStatus: "unofficial",
        copyrightStart:    "2019",
        //shortName:  "json-ld-bp",
        edDraftURI: "https://w3c.github.io/json-ld-bp/",
        github: {
          repoURL: "https://github.com/w3c/json-ld-bp/",
          branch: "main"
        },
        editors: [{
          name: "Gregg Kellogg",
          url: "http://greggkellogg.net/",
          w3cid: "44770"
        }],
        //previousMaturity: "NOTE",
        //previousPublishDate: "1977-03-15",
        github: {
          repoURL: "https://github.com/w3c/json-ld-bp/",
          branch: "main"
        },
        doJsonLd:   true,
        group: "wg/json-ld",
        wgPublicList: "public-json-ld-wg",
        maxTocLevel:  4,
        noRecTrack:   true
      };

**Comment by marcoscaceres** [comment]
Yeah, that's definitely a bug... For some reason, the W3C's header template is being loaded and adding the Latest Version... it shouldn't do that for unofficial drafts :(

**Comment by gkellogg** [comment]
It would be nice for this issue to get some love; it seems like a rather serious issue at the heart of ReSpec, even if it might not be apparent in other usages.

**Comment by sidvishnoi** [comment]
I'll try to find some time in week of 21st. Away at present.

**Comment by TallTed** [comment]
Maybe soon after TPAC 2023? Since I can't imagine it'll get touched before then...
